app/services/activitypub/fetch_remote_status_service.rb

```diff
@@ -11,7 +11,7 @@ def call(uri, id: true, prefetched_body: nil)
               body_to_json(prefetched_body)
             end
 
-    return unless expected_type? && supported_context?
+    return unless supported_context? && expected_type?
```

Comment:
I think on balance I would prefer if the nilcheck was more explicit here, as a separate line `return if @json.nil?`

Reply:
I'm pretty sure context check must go first always because if the context is wrong then the type doesn't matter.